services/QuillLMS/spec/models/activity_spec.rb

```diff
@@ -877,4 +877,38 @@
       it { expect(subject).to eq(activity.data) }
     end
   end
+
+  describe 'diagnostic repeatable behavior' do
+    context 'when activity is diagnostic' do
+      let(:diagnostic_classification) { create(:diagnostic) }
+
+      it 'defaults repeatable to false on new record' do
+        diagnostic_activity = build(:activity, classification: diagnostic_classification)
```

Comment:
I'd move this `build` into a `let` since it's used in every `it` block in this `context`:
```ruby
let(:diagnostic_classification) { create(:diagnostic) }
let(:diagnostic_activity) { build(:activity, classification: diagnostic_classification) }
```